### PR TITLE
add serpbear

### PR DESCRIPTION
**Pull request by jonas-hoebenreich**
see https://github.com/caprover/one-click-apps/issues/878

https://github.com/towfiqi/serpbear
https://hub.docker.com/r/towfiqi/serpbear

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/serpbear.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        restart: always
+        image: 'towfiqi/serpbear:$$cap_serpbear_version'
+        environment:
+            NEXT_PUBLIC_BASE_URL: 'http://$$cap_appname.$$cap_root_domain'
```
Comment (githubsaturn) [inline public/v4/apps/serpbear.yml:7]: Where is `NEXT_PUBLIC_BASE_URL` being used?

Comment (githubsaturn) [reply public/v4/apps/serpbear.yml:7]: Ah I think you meant to use `NEXT_PUBLIC_APP_URL`

File: public/v4/apps/serpbear.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            PASSWORD: $$cap_serpbear_password
+            SECRET: $$cap_serpbear_secret
+            APIKEY: $$cap_serpbear_apikey
+            SESSION_DURATION: $$cap_serpbear_session
+        caproverExtra:
+            containerHttpPort: '3000'
+        volumes:
+            - '$$cap_appname-serpbear-data:/app/data/serpbear'
+        networks:
+            - serpbear-net
```
Comment (githubsaturn) [inline public/v4/apps/serpbear.yml:18]: This is not used by CapRover, you can remove.

File: public/v4/apps/serpbear.yml (hunk 1 above)
Comment (jonas-hoebenreich) [reply public/v4/apps/serpbear.yml:7]: Good catch, thanks for noticing

File: public/v4/apps/serpbear.yml (hunk 2 above)
Comment (jonas-hoebenreich) [reply public/v4/apps/serpbear.yml:18]: ok 👌

**Comment by githubsaturn** [comment]
Thanks!

**Comment by uhlhosting** [comment]
@githubsaturn first of all this is not official supported app, since the person who submitted it is not the owner. Some very important feature of the app does not seem to work. And even the owner does not seem to find a solution to it. Yet adding new apps should not get the official app badge, unless submitted and maintained by their makers. Hope this makes sense.

**Comment by githubsaturn** [comment]
> first of all this is not official supported app

@uhlhosting  - where do you get this? As far as I can see, it's the image that is linked in the official repository. What makes you think that it's not the official image?

**Comment by uhlhosting** [comment]
@githubsaturn my bad, the image uses the official app indeed, yet I tough official apps are the ones submitted and maintained by their makers.

**Pull request merged**